src/main/docs/guide/querying/specifications.adoc

```diff
@@ -0,0 +1,27 @@
+Currently supported only by JPA runtime.
```

Comment:
Maybe make this prefixed with `NOTE: ` and move under the introductory paragraph below